### PR TITLE
Solved the white screen issue

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
It was caused because of screen util package not configured correctly.

### CODE REVIEW

File: lib/widgets/splash_widget.dart [hunk 1]
```diff
@@ -3,14 +3,13 @@ import 'package:flutter_screenutil/flutter_screenutil.dart';
...
-
-  @override
-  Widget build(BuildContext context) {
-    return SizedBox(
-        height: 1.sh,
-        width: 1.sw,
-        child: SvgPicture.asset(kSvgSplash, fit: BoxFit.cover));
-  }
-}
+// class SplashWidget extends StatelessWidget {
```
Comment (mustafapylons) [inline lib/widgets/splash_widget.dart]: Do we need the dead code for testing purpose or similar?

**Review by mustafapylons — COMMENTED** [review]
One small issue, otherwise code is good.

File: lib/widgets/splash_widget.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply lib/widgets/splash_widget.dart]: No let me remove it.

**Review by kiroj305 — APPROVED** [review]
Looks good to me. I removed the unused widget.

**Comment by kiroj305** [comment]
@mustafapylons , I'd merge it if you approve. I wanna create a new pr based on merging this commit first.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Comment by mustafapylons** [comment]
@Tiny-Angel305 approved on my side.

**Pull request merged**